packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -153,26 +168,28 @@ private MediaSource buildMediaSource(
     }
   }
 
-  private void setupVideoPlayer(
-      EventChannel eventChannel, TextureRegistry.SurfaceTextureEntry textureEntry) {
+  private void setupVideoPlayer(ExoPlayer exoPlayer, QueuingEventSink eventSink) {
```

Comment:
While you're touching this, could you fix the name to `setUpVideoPlayer`? (This mistake is widespread in the plugin code, unfortunately; I'm trying to fix it incrementally whenever we find it.)